src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from 'redux';
+import BookReducer from './book';
+
+const store = createStore(BookReducer);
+console.log('store state from reducers', store.getState());
```

Comment:
remove the console and use  `combinedReducers` to  combines app specific reducers